gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -78,25 +80,34 @@
 @AndroidEntryPoint
 public class HomeScreenFragment extends AbstractFragment
     implements BackPressListener, OnNavigationItemSelectedListener, OnGlobalLayoutListener {
+
   // TODO: It's not obvious which feature are in HomeScreen vs MapContainer; make this more
   // intuitive.
   private static final float COLLAPSED_MAP_ASPECT_RATIO = 3.0f / 2.0f;
 
-  @Inject AddFeatureDialogFragment addFeatureDialogFragment;
-  @Inject AuthenticationManager authenticationManager;
-  @Inject Schedulers schedulers;
-  @Inject Navigator navigator;
-  @Inject EphemeralPopups popups;
+  @Inject
+  AddFeatureDialogFragment addFeatureDialogFragment;
+  @Inject
+  AuthenticationManager authenticationManager;
+  @Inject
+  Schedulers schedulers;
+  @Inject
+  Navigator navigator;
+  @Inject
+  EphemeralPopups popups;
   MapContainerViewModel mapContainerViewModel;
 
-  @Nullable private ProgressDialog progressDialog;
+  @Nullable
+  private ProgressDialog progressDialog;
   private HomeScreenViewModel viewModel;
   private MapContainerFragment mapContainerFragment;
   private BottomSheetBehavior<View> bottomSheetBehavior;
   private ProjectSelectorDialogFragment projectSelectorDialogFragment;
   private ProjectSelectorViewModel projectSelectorViewModel;
   private List<Project> projects = Collections.emptyList();
   private HomeScreenFragBinding binding;
+  // Will be removed later have just added for some testing
+  private final ArrayList<Point> polygonPoints = new ArrayList<>();
```

Comment:
please delete if added for testing ?

Reply:
+1. Or add `TODO(#bug):` if this must be submitted.